examples/after-reboot-check.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: example-after-reboot-check
+  namespace: kube-system
+spec:
+  template:
+    metadata:
+      labels:
+        app: example-after-reboot-check
+    spec:
+      nodeSelector:
+        container-linux-update.v1.coreos.com/after-reboot: "true"
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: example-after-reboot-check
+        image: quay.io/stephen_demos/kube-annotate:latest
```

Comment:
How much effort is it to use the hyperkube image (which contains kubectl) and service account or host kubeconfig and just run ``kubectl annotation node blah`?